File: app/src/main/java/home/oleg/placesnearme/models/Category.java
package home.oleg.placesnearme.models;

/**
 * Created by Oleg on 12.04.2016.
 */
public class Category {

    private String id;

    private String name;

    private String pluralName;

    private String shortName;

    private String[] parents;

    private boolean primary;

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getPluralName() {
        return pluralName;
    }

    public String[] getParents() {
        return parents;
    }

    public boolean isPrimary() {
        return primary;
    }

    public String getShortName() {
        return shortName;
    }

}

File: core-network/src/main/java/home/oleg/placesnearme/corenetwork/models/Category.kt
package home.oleg.placesnearme.corenetwork.models

data class Category(val id: String? = null,
                    val name: String? = null,
                    val pluralName: String? = null,
                    val primary: Boolean,
                    val shortName: String? = null,
                    val icon: CategoryIcon? = null)
